ci/build_scripts/build.sh

```diff
@@ -80,6 +80,8 @@ ARCH=
 TARGET=()
 BUILD_OPTIONS=()
 BUILD=1
+INCLUDE_TEST_PACKAGES=1
+INCLUDE_DEPRECATED_PACKAGES=1
```

Comment:
How about excluding these packages by default as these are useful only in rare occasions?

Reply:
I wanted to keep with the current behaviour as the tests required the deprecated packages still. So I would rather remove the deprecated packages and tests in a different PR.